feature/amazon/src/main/AndroidManifest.xml

```diff
@@ -3,13 +3,19 @@
         package="com.revenuecat.purchases.amazon">
 
     <application>
-        <receiver android:name = "com.amazon.device.iap.ResponseReceiver"
-                android:permission = "com.amazon.inapp.purchasing.Permission.NOTIFY"
-                android:exported = "true">
+        <receiver
+                android:name="com.amazon.device.iap.ResponseReceiver"
+                android:exported="true"
+                android:permission="com.amazon.inapp.purchasing.Permission.NOTIFY">
             <intent-filter>
-                <action android:name = "com.amazon.inapp.purchasing.NOTIFY" />
+                <action android:name="com.amazon.inapp.purchasing.NOTIFY" />
             </intent-filter>
         </receiver>
+        <activity
+                android:name=".ProxyAmazonBillingActivity"
+                android:configChanges="keyboard|keyboardHidden|screenLayout|screenSize|orientation"
+                android:exported="false"
```

Comment:
`exported` needs to be explicitely set when targeting Android 12 apps, otherwise the installation fails https://developer.android.com/about/versions/12/behavior-changes-12#exported